docs/guide/http_cache.md

```diff
@@ -326,6 +336,8 @@ fastcgi_param SYMFONY_TRUSTED_PROXIES "193.22.44.22";
 
 Secondly we need to tell eZ Platform to change to use http based purge client *(specifically FosHttpCache Varnish purge client is used)*, and specify url Varnish can be reached on:
 
+Since eZ Platform 2.x, this following configuration is not required as eZ will read the Environment Variables set above.
```

Comment:
As this will only go to 2.0+ version of the doc, let's drop `Since eZ Platform 2.x,` and just do `The following configuration is not required as eZ Platform will read the environment variables set above.`

(we do not use "eZ" by itself in the doc to refer to eZ Platform)